<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertCoorDlg</class>
 <widget class="QDialog" name="ConvertCoorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EPSG:4326&lt;=&gt;EPSG:3857(墨卡托投影)转换</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>EPSG:4326坐标</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="m_lonEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="m_latEdit"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="convertEPSG3857Btn">
     <property name="text">
      <string>转EPSG:3857</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>EPSG:3857坐标</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="m_xEdit"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="m_yEdit"/>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="convertEPSG4326Btn">
     <property name="text">
      <string>转EPSG:4326</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
